arbnode/batch_poster.go

```diff
@@ -800,10 +800,18 @@ func (s *batchSegments) recompressAll() error {
 func (s *batchSegments) testForOverflow(isHeader bool) (bool, error) {
 	// we've reached the max decompressed size
 	if s.totalUncompressedSize > arbstate.MaxDecompressedLen {
+		log.Info("Batch overflow: max decompressed length exceeded",
```

Comment:
Batch overflow seems like an error, but this is an expected case. Maybe "batch full" or something similar?

Reply:
Agreed.